common/on_action/schemes/wc_stasis_trap_on_actions.txt

```diff
@@ -0,0 +1,84 @@
+﻿stasis_trap_setup = {
+	first_valid_on_action = {
+		standard_stasis_trap_method_setup
+	}
+}
+
+standard_stasis_trap_method_setup = {
+	random_events = {
+		100 = stasis_trap_outcome.1001 # 
+	}
+}
+
+
+stasis_trap_succeeded = {
+	# Picks out the correct success event depending on setup.
+	trigger = {
+		exists = scope:scheme
+	}
+	first_valid = {
+		stasis_trap_outcome.2001 #Seized Roadside
+	}
+}
+
+stasis_trap_failed = {
+	# Picks out the correct failure event depending on setup/save.
+	trigger = {
+		exists = scope:scheme
+	}
+	first_valid = {
+		stasis_trap_outcome.4001 #Seized Roadside
+	}
+}
+
+stasis_trap_ongoing = {
```

Comment:
I don't see any new events here so maybe you can remove it and refer to abduction or murder ongoing events?